docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -134,10 +134,28 @@ When you deploy the model, it is allocated to all available {ml} nodes. The
 model is loaded into memory in a native process that encapsulates `libtorch`,
 which is the underlying machine learning library of PyTorch.
 
-TIP: You can set additional deployment options that affect the performance of
-the model. For example, specify the number of allocations and the threads per
-allocation to optimize the throughput and speed of inference requests in your
-production environment. For more information about these options, refer to the
+You can set additional deployment options - the number of allocations and the 
+threads per allocation - that affect the performance of the model.
+
+Model allocations are independent units of work for NLP tasks. Throughput can be 
+scaled by adding more allocations to the deployment, it increases the number of 
+{infer} requests that can be performed in parallel. Allocations are distributed 
+to different {ml} nodes where possible. In some situations, however, multiple 
```

Comment:
`Allocations are distributed to different {ml} nodes where possible`. That statement is not true. 

I suggest reusing this sentence from the start API reference: ```Model allocations are distributed across machine learning nodes. All allocations assigned to a node share the same copy of the model in memory.```